front/app/containers/ProjectsShowPage/timeline/Ideas.tsx

```diff
@@ -81,27 +80,28 @@ const IdeasContainer = ({ projectId, phase, className }: InnerProps) => {
     showViewToggle: true,
     defaultView: phase.attributes.presentation_mode,
   };
+  const useSidebarFilters = config.showIdeaFilters === true;
```

Comment:
Normally the `use` word in the begging is used for React hooks. We should rename this.